### PR TITLE
feature/OSINT-273

### PR DESCRIPTION
**Pull request by ahrufox**
# [OSINT-273] Maigret 구현
## 🔑 Key Change 
- In tool_maigret.py, run_maigret and check_maigret works.
- Add run.input_value. Name can be changed(if you want)
- Add some exception code in whois when working maigret.
- whois will not response when request getToolState... for now....

## 🖼️ Screenshots (if necessary)
nope

## 🙏 To Reviewers
🧑‍🤝‍🧑 
@oo7-0310 Hi there!
@D7MeKz DB check plz
@SHJ99 edited getToolState check plz

[OSINT-273]: https://bob-transfer.atlassian.net/browse/OSINT-273?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: server/osint/tools/lib/tool_maigret.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        message = f'Run maigret failed. Username is {run.input_value}. Return code: {e}'
+        run.status = 'error'
+        return message
+
+    run.save()
+    return run.run_id
+
+
+def check_maigret(run):
+    if run.status == 'initiate':
```
Comment (D7MeKz) [inline server/osint/tools/lib/tool_maigret.py:21]: - Is **initiate** in run.status default value?

**Comment by SHJ99** [comment]
I checked, but I can't figure out why it doesn't work... Why?

**Comment by ahrufox** [comment]
> I checked, but I can't figure out why it doesn't work... Why?

Error code?

**Review by D7MeKz — APPROVED** [review]

File: server/osint/tools/lib/tool_maigret.py (hunk 1 above)
Comment (ahrufox) [reply server/osint/tools/lib/tool_maigret.py:21]: before: running -> completed / error
after: initiate -> running -> completed / error

**Pull request merged**